docs/guides/first-device-level-experiment.mdx

```diff
@@ -3,62 +3,149 @@ sidebar_label: Device-level Experiments
 title: Build your first Device-level Experiment
 ---
 
-In cases where you're unable to establish the user's identity and are unable to leverage a user ID as the unit of randomization for your experiment,
-Statsig's _device-level_ experiments enable you to design experiments using a Statsig-generated stable ID, saved on the user's device in local storage.
+When a user's identity is unknown and a user ID can’t be used for randomization, Statsig's _device-level_ experiments allow you to use a stable ID stored on the user's device. This stable ID is automatically generated by Statsig and stored in the user's local storage, allowing you to continue running experiments seamlessly without relying on user logins.
 
-Setting up a _device-level_ experiment is the same as creating a traditional _user-level_ experiment except that you simply have to set the ID type on the experiment to Stable ID when creating the experiment.  You can pass your user object with or without a user ID when initializing the client SDK - the SDK will generate a stable ID for that user to use for evaluating these experiments.
+### How Device-level Experiments Work
+- A **Stable ID** is a unique identifier generated by Statsig and saved in the device's local storage.
+- This ID is used to evaluate experiments when a user ID is unavailable.
```

Comment:
its not used when a userid is unavailable - its used when the experiments ID type is "stableID" instead of userID